apps/portal-front/src/components/unenroll/octi/index.tsx

```diff
@@ -0,0 +1,129 @@
+import {
+  CanUnenrollOCTIInstanceFragment,
+  UnenrollOCTIInstance,
+} from '@/components/enroll/enroll.graphql';
+import { EnrollStateLayout } from '@/components/enroll/state/layout';
+import Loader from '@/components/loader';
+import { UnenrollOCTIConfirm } from '@/components/unenroll/octi/confirm';
+import { UnenrollOCTIInstanceNotEnrolled } from '@/components/unenroll/octi/instance-not-enrolled';
+import { UnenrollOCTIMissingCapability } from '@/components/unenroll/octi/missing-capability';
+import useMountingLoader from '@/hooks/useMountingLoader';
+import { enrollCanUnenrollOCTIInstanceFragment$key } from '@generated/enrollCanUnenrollOCTIInstanceFragment.graphql';
+import EnrollCanUnenrollOCTIInstanceQueryGraphql, {
+  enrollCanUnenrollOCTIInstanceQuery,
+} from '@generated/enrollCanUnenrollOCTIInstanceQuery.graphql';
+import { enrollUnenrollOCTIInstanceMutation } from '@generated/enrollUnenrollOCTIInstanceMutation.graphql';
+import UserListOrganizationAdministratorsQueryGraphql, {
+  userListOrganizationAdministratorsQuery,
+} from '@generated/userListOrganizationAdministratorsQuery.graphql';
+import { toast } from 'filigran-ui/clients';
+import { useTranslations } from 'next-intl';
+import React, { useState } from 'react';
+import {
+  PreloadedQuery,
+  useFragment,
+  useMutation,
+  usePreloadedQuery,
+  useQueryLoader,
+} from 'react-relay';
+
+interface Props {
+  platformId: string;
+  queryRef: PreloadedQuery<enrollCanUnenrollOCTIInstanceQuery>;
+}
+
+type UnenrollmentStatus = 'idle' | 'succeeded' | 'failed';
+
+export const UnenrollOCTI: React.FC<Props> = ({ queryRef, platformId }) => {
+  const t = useTranslations();
+  const canUnenrollPreloadedQuery =
+    usePreloadedQuery<enrollCanUnenrollOCTIInstanceQuery>(
+      EnrollCanUnenrollOCTIInstanceQueryGraphql,
+      queryRef
+    );
+
+  const { instance } = useFragment<enrollCanUnenrollOCTIInstanceFragment$key>(
```

Comment:
Linked to my previous suggestion.
```suggestion
  const instance = useFragment<enrollCanUnenrollOCTIInstanceFragment$key>(
```